src/assets/help-wanted.json

```diff
@@ -44,5 +44,115 @@
       "agency_id": "GSA",
       "license": "CC0-1.0"
     }
+    {
+      "id": 3,
+      "title": "Allow user to view an item from the left pane while editing",
+      "description": "Issue Description:\nWhen editing, a very common observation is users single-clicking an object in the left pane (both browse tree and search) and being taken out of Edit mode, or being warned with the 'Unsaved changes' dialog. What they want is to be able to see an object before adding it to a thing they're currently editing.\n\nProposed solution:\n\nSingle-clicking in the left pane during Edit mode displays the clicked object in an overlay without taking the user out of Edit mode.\nAll normal 'view' functionality (like plot pan/zoom) would be available.\nThere should be a 'Add This Item' button available in the overlay as well that would add the item being viewed to the object being edited, then closes the overlay.",
+      "updated": "2017-12-13",
+      "languages": [
+        "CSS",
+        "HTML",
+        "JavaScript"
+      ],
+      "skill": "Intermediate",
+      "effort": "Medium",
+      "type": "Enhancement",
+      "impact": "Open Government",
+      "projectURL": "https://github.com/nasa/openmct",
+      "issueURL": "https://github.com/nasa/openmct/issues/928",
+      "featured": true,
+      "active": true,
+      "popular": true,
+      "agency_id": "NASA",
+      "license": "Apache License, Version 2.0"
+    }
```

Comment:
Very minor typo, but it'll break compilation.  There should be a comma after the curly brace for this and other lines that end objects.  `}` should change to `},`